implementation/endpoints/include/netlink_connector.hpp

```diff
@@ -161,9 +161,9 @@ class netlink_connector : public std::enable_shared_from_this<netlink_connector>
     bool has_address(struct ifaddrmsg * ifa_struct,
             size_t length,
             const unsigned int address);
-    void send_ifa_request();
-    void send_ifi_request();
-    void send_rt_request();
+    void send_ifa_request(unsigned int retry = 0);
+    void send_ifi_request(unsigned int retry = 0);
+    void send_rt_request(unsigned int retry = 0);
```

Comment:
just to be coherent with the rest of the code.
```suggestion
    void send_ifa_request(std::uint32_t retry = 0);
    void send_ifi_request(std::uint32_t retry = 0);
    void send_rt_request(std::uint32_t retry = 0);
```